src/integrations/admin/indexation-integration.php

```diff
@@ -264,6 +265,10 @@ protected function get_total_unindexed() {
 	 * @return bool True if hidden.
 	 */
 	protected function is_indexation_warning_hidden() {
+		if ( $this->has_indexation_reason() ) {
```

Comment:
Before we would show the general one before the specific one.
Now this seems almost reversed. Should this be reflected in the add_action if/else in enqueue_scripts?